style.css

```diff
@@ -29,13 +30,14 @@ Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, feature
  * Link styles
  * https://github.com/WordPress/gutenberg/issues/42319
  */
+
 a {
-	text-decoration-thickness: .0625em !important;
-	text-underline-offset: .15em;
+	text-decoration-thickness: 0.0625em !important;
```

Comment:
Why does this need `!important`? The linked Gutenberg issue is about setting custom values for underline styles such as `text-decoration-thickness`.

Reply:
This PR removes the `!important` now.